bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -430,6 +430,10 @@ spec:
         - name: "NO_PROXY"
           value: "{{ .NO_PROXY}}"
 {{ end }}
+        {{ if .IsLooseUDNIsolationEnabled }}
+        - name: UDN_ISOLATION_MODE
+          value: "loose"
+        {{ end }}
```

Comment:
I don't think this matches what https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5276/commits/3dd614922518bb513795f9c1f7951f9e13563ddc expects. The setting there is called
`ROUTED_UDN_ISOLATION` instead of `UDN_ISOLATION_MODE`. Also the values it takes are `Disabled` or `Enabled` not `loose`.

This should preferably be passed as a command line argument in `script-lib` rather than an environment variable for consistency.

I think `UDN_ISOLATION_MODE` is fine, I would probably use `Strict` and `Loose` for values rather than Disabled` or `Enabled`.